airflow-core/src/airflow/utils/db.py

```diff
@@ -125,9 +125,30 @@ def add_default_pool_if_not_exists(session: Session = NEW_SESSION):
 @provide_session
 def create_default_connections(session: Session = NEW_SESSION):
     """Create default Airflow connections."""
+    conns = get_default_connections()
+
+    for c in conns:
+        merge_conn(c, session)
+
+
+def create_default_connections_for_tests():
+    """
+    Create default Airflow connections for tests.
+
+    For testing purposes, we do not need to have the connections setup in the database, using environment
+    variables instead would provide better lookup speeds and is easier too.
+    """
+    conns = get_default_connections()
+
+    for c in conns:
+        envvar = f"AIRFLOW_CONN_{c.conn_id.upper()}"
+        os.environ[envvar] = c.as_json()
```

Comment:
Worth moving this in tests - `devel-common/src/tests_common/test_utils/db.py`?

Reply:
Of course! That would only make sense